client/my-sites/sites/index.jsx

```diff
@@ -78,6 +78,17 @@ export class Sites extends Component {
 				break;
 		}
 
+		if ( path === 'page' ) {
+			return i18n.translate( 'Select a site to start writing', {
+				args: {
+					path: path,
+				},
+				components: {
+					strong: <strong />,
+				},
+			} );
+		}
```

Comment:
We don't need the options object as the translation string doesn't have any variables to be replaced:

```js
return i18n.translate( 'Select a site to start writing' );
```